src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const copyState = Object.assign({}, state);
```

Comment:
@it's better to use destructuring instead of Object.assign(), because you can reduce you code that way
 ```suggestion
  const copyState = { ...state };
```